README.md

```diff
@@ -9,3 +9,11 @@ This repository is the home of the following packages:
 -   [`@eslint/object-schema`](packages/object-schema)
 -   [`@eslint/config-array`](packages/config-array)
 -   [`@eslint/compat`](packages/compat)
+
+## Sponsors
+
+The following companies, organizations, and individuals support ESLint's ongoing maintenance and development. [Become a Sponsor](https://opencollective.com/eslint) to get your logo on our README and website.
```

Comment:
```suggestion
The following companies, organizations, and individuals support ESLint's ongoing maintenance and development. [Become a Sponsor](https://eslint.org/donate) to get your logo on our README and website.
```

Reply:
changed all occurences. Shall I send a PR to update the same in eslint/eslint repo as well?